transformer_engine/common/comm_gemm_overlap/comm_gemm_overlap.cpp

```diff
@@ -90,13 +90,25 @@ CommOverlapCore::CommOverlapCore(int myrank, int numranks, int mylocal, int numl
   cudaEventCreateWithFlags(&_stop_compute, 0);
   cudaEventCreateWithFlags(&_start_comm, 0);
   cudaEventCreateWithFlags(&_stop_comm, 0);
+
+  //Managing launch ordering to maximize comm-comp overlap for the case of using CUDA_DEVICE_MAX_CONNECTIONS>1
```

Comment:
`Managing launch ordering to maximize comm-comp overlap for the case of using CUDA_DEVICE_MAX_CONNECTIONS>1`
to
`Defining the launcher order between the communication and GEMM kernels using Fast Dependent Launch when CUDA_DEVICE_MAX_CONNECTIONS>1. The event is used to schedule the communication kernel before the GEMM. This is needed only for Hopper, which uses persistent CTA execution.`